### ISSUE TITLE
Counter Vanilla example broken

### ISSUE THREAD

**Issue by nonissue** [reduxjs/redux-toolkit#297]
The counter vanilla example does not work when viewed on codesandbox or when downloaded locally (macos, tried Chrome/Firefox/Safari). Page loads, but an error message about 'require' being undefined is thrown, buttons don't update or change the state in any way.

Locally, I managed to fix the issue by changing:

```
<script src="https://unpkg.com/@reduxjs/toolkit@latest/dist/redux-toolkit.umd.js"></script>
```

to 

```
<script src="https://unpkg.com/@reduxjs/toolkit@latest/dist/redux-toolkit.umd.min.js"></script>
```

Not sure if the codesandbox demo is supposed to be interactive, but I feel like it should be. The same fix didn't work there though.

Hopefully this is all clear, let me know if any further info is needed.

**Comment by phryneas** [comment]
Hey, thank you for this report. This is indeed a bundling issue.

https://unpkg.com/@reduxjs/toolkit@1.2.1/dist/redux-toolkit.umd.js should not contain 
```js

      /* PROD_START_REMOVE_UMD */
      var createImmutableStateInvariantMiddleware = require('redux-immutable-state-invariant')["default"];

      var immutableOptions = {};

      if (!isBoolean(immutableCheck)) {
        immutableOptions = immutableCheck;
      }

      middlewareArray.unshift(createImmutableStateInvariantMiddleware(immutableOptions));
      /* PROD_STOP_REMOVE_UMD */
```

This seems to be a regression due to #280. I'll ping the author over there.

**Comment by alex-ketch** [comment]
@phryneas Hi apologies for the lack of communication over the holidays.
Will take a look at this tonight

**Comment by markerikson** [comment]
This is fixed now that 1.2.2 is out.
